src/main/java/persistence/sql/entity/EntityColumns.java

```diff
@@ -0,0 +1,37 @@
+package persistence.sql.entity;
+
+import jakarta.persistence.Transient;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class EntityColumns {
+    private final List<EntityColumn> columns;
+
+    public EntityColumns(List<EntityColumn> columns) {
+        this.columns = columns;
+    }
+
+    public static EntityColumns from(Class<?> clazz) {
+        List<EntityColumn> collect = Arrays.stream(clazz.getDeclaredFields())
+                .filter(field -> !field.isAnnotationPresent(Transient.class))
+                .map(EntityColumn::from)
+                .collect(Collectors.toList());
+        return new EntityColumns(collect);
```

Comment:
사소하지만 아래와 같이 `collectingAndThen` 을 사용해주시면 깔끔하게 구현할 수 있어요 😄 
```suggestion
    return Arrays.stream(clazz.getDeclaredFields())
                .filter(field -> !field.isAnnotationPresent(Transient.class))
                .map(EntityColumn::from)
                .collect(Collectors.collectingAndThen(Collectors.toList(), EntityColumns::new));
```

Reply:
collecting을하고 메서드를 하나 더 호출하는 메서드가 있었군요 감사합니다 👍